service/create/service.go

```diff
@@ -942,6 +966,17 @@ func (s *Service) processCluster(cluster awstpr.CustomObject) error {
 
 	s.logger.Log("info", fmt.Sprintf("processed autoscaling group stack '%s'", key.AutoScalingGroupName(cluster, prefixWorker)))
 
+	if legacyASG {
+		cfAsg, err := s.findCFASG(clients.AutoScaling, key.AutoScalingGroupName(cluster, prefixWorker), key.ClusterID(cluster))
+		if err != nil {
+			return microerror.Maskf(executionFailedError, fmt.Sprintf("could find CF ASG: '#%v'", err))
```

Comment:
I think the error should be "could not find".

Reply:
It is also that `Maskf` does the formatting already. No need to use `fmt` here. <3

Reply:
Good catch! fixed